rasa/shared/nlu/training_data/message.py

```diff
@@ -4,6 +4,7 @@
 import copy
 
 import rasa.shared.utils.io
+from rasa.nlu.constants import TOKENS_NAMES
```

Comment:
@joejuzl this import actually causes the test checking that the `shared` package is independent to fail.
I could potentially add a constant for TEXT_TOKENS in `rasa/shared/nlu/constants.py`?